docker/web/Dockerfile

```diff
@@ -127,19 +127,14 @@ RUN npm install -g npm@8.1.0
 # Install composer
 RUN curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer --2
 
-## Install Drush Launcher (runs local drush instances)
-RUN curl -sL -o /usr/local/bin/drush https://github.com/drush-ops/drush-launcher/releases/download/0.10.1/drush.phar \
-  && chmod +x /usr/local/bin/drush
+# Verify php and composer installs.
+RUN php --version; composer --version;
 
-## Install Drupal console
-RUN curl https://drupalconsole.com/installer -L -o /usr/local/bin/drupal \
-  && chmod +x /usr/local/bin/drupal
+# Make vendor/bin versions of drush, blt, etc. available.
+ENV PATH /var/www/vendor/bin/:$PATH
```

Comment:
Can you just add this to the end of the path so it matches the CI workflow?

Reply:
Done